libraries/helpers.rb

```diff
@@ -1,6 +1,13 @@
 module OSLMatrix
   module Cookbook
     module Helpers
+      class ::Hash
```

Comment:
Can you please add a comment for where this came from and what this does?

Reply:
All configuration settings for anything yaml related is done programically, with hashes. But I've came across a problem when trying to merge default settings with custom additions. Ruby doesn't come with a proper way of recursively merging Hashes, I'll add in an attribution and explanation for this addition.